lib/cluster/vpc.rb

```diff
@@ -231,6 +231,11 @@ def self.create_peer_connection(vpc, peer_vpc)
         peer_vpc_name = peer_vpc.id
       end
 
+      ec2_client.wait_until(
+        :vpc_peering_connection_exists,
+        vpc_peering_connection_ids: [connection_id]
```

Comment:
Would it be worth adding a link to the AWS Ruby SDK docs in a  REAME? I ask because when I Google stuff like https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/using-waiters.html it looks different.

Reply:
I don't remember exactly what bothered me about this area. It was related to a comment in the change, that I don't see anymore, so I'm resolving this comment.

Reply:
Turns out there were some links already but they were pointing to v2. I've updated them.